fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -207,6 +207,13 @@ def self.user_defined_itms_path?
       return FastlaneCore::Env.truthy?("FASTLANE_ITUNES_TRANSPORTER_PATH")
     end
 
+    def self.path_name_itmsp
+      unless @name_itms
+        @name_itms = ENV["FASTLANE_ITUNES_TRANSPORTER_PATH_ITMSP"] || "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx".gsub("x") { rand(16).to_s(16) }
+      end
```

Comment:
```suggestiont
       @name_itms ||= ENV["FASTLANE_ITUNES_TRANSPORTER_PATH_ITMSP"] || SecureRandom.uuid
```

Thoughts on this? 😊

Reply:
Thank you so much for your comment @joshdholtz, it's really amazing!

I include in the path the **app_id** again,  this resolve your comment 😃?